pyccel/stdlib/ndarrays/ndarrays.h

```diff
@@ -9,6 +9,7 @@
 # include <complex.h>
 # include <stdbool.h>
 # include <stdint.h>
+# include <cblas.h>
```

Comment:
Why do you need to import this here rather than adding it to the imports of the generated file?